drivers/clock_control/Kconfig.it8xxx2

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2022 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_ITE_IT8XXX2_CLOCK := ite,it8xxx2-ecpm
+
+config CLOCK_CONTROL_IT8XXX2
+	bool "ITE IT8XXX2 clock controller driver"
+	depends on SOC_FAMILY_RISCV_ITE
```

Comment:
Should this use `SOC_SERIES_RISCV32_IT8XXX2` instead, since it's a IT8xxx2 driver?

Reply:
@tari 

Yes, sir. You're right, I should use this `SOC_SERIES_RISCV32_IT8XXX2` instead. Now it's updated. Appreciative of your guidance.